test/minikube/minikube_base_tde_test.go

```diff
@@ -0,0 +1,213 @@
+// +build enterprise
```

Comment:
Does this actually require EE, i.e. do these tests start multiple SMs? Or do we just not want to run this test in GitHub CI because it uses unreleased functionality? I introduced the NUODB_DEV environment variable for the latter case.

Reply:
Good point! The test doesn't require enterprise functionality so we could leverage of `NUODB_DEV` variable.